plugins/modules/source_control/gitlab/gitlab_user.py

```diff
@@ -412,13 +458,50 @@ def createUser(self, arguments):
         if self._module.check_mode:
             return True
 
+        if 'identities' in arguments:
```

Comment:
```suggestion
        identities = None
        if 'identities' in arguments:
```

Reply:
identities is an empty list when not specified